### PR TITLE
add repo package flink2

### PR DESCRIPTION
**Pull request by luoomu**
add flink 1.3.0-1.0 to packages

### CODE REVIEW

**Comment by OhRobin** [comment]
@luoomu Could you please put a PR with an updated example here: https://github.com/dcos/examples ?

**Comment by joerg84** [comment]
@OhRobin No

File: repo/packages/F/flink/2/resource.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "flink": "mesosphere/dcos-flink:1.3.0-1.0"
```
Comment (joerg84) [inline repo/packages/F/flink/2/resource.json]: You are missing the Scala 2.11 image here, I will add it.

**Comment by joerg84** [comment]
Updated with https://github.com/mesosphere/universe/pull/1252

**Pull request closed without merging**